src/createPipe.ts

```diff
@@ -3,14 +3,17 @@ import { pipe } from "./pipe";
 /**
  * Creates a data-last pipe function. First function must be always annotated. Other functions are automatically inferred.
  *
+ * ! **DEPRECATED**: Use `R.piped(op1, op2, op3)`. Will be removed in V2!
```

Comment:
**todo:** if we're renaming `createPipe`, we should change the mentions in the README and in the docs index page

Reply:
Thanks for catching that! We're thinking of rewriting the readme as part of v2 so we will probably take a closer look at it soon.